docs/spectator/core/meters/timer.md

```diff
@@ -29,6 +30,127 @@ Long Task Timer:
 
 ![Long Task Timer](../../../images/duration_timer.png)
 
+[Percentile Timer]: percentile-timer.md
+## Querying
+
+!!! Note
+    Timers report summarized statistics about the measurements for a time window
+    including the `totalTime`, `count`, `max` and `totalOfSquares`. If you were to simply query for
+    the name of your timer via
+
+    ```
+    nf.cluster,foo,:eq,name,http.req.latency,:eq,:and
+    ```
+
+    you would get a nonsense value that is the sum of the reported statistics.
+
+When querying the results of a timer, use one of the ASL functions below to generate
+a useful response.
+
+### Average Measurement (:dist-avg)
+
+For [Timer] and Distribution Summary metrics, the `totalTime` (timers) /`totalAmount` (distributions)
+and `count` are collected each time a measurement is taken. If this technique was applied to a
+request latency metric, then you would have the average latency per request for an arbitrary
+grouping. These types of metrics have an explicit count based on activity. To get an average
+per measurement manually:
+
+```
```

Comment:
Using the `@@@ atlas-stacklang` directive is preferable for these sort of snippets so all operators are linked back to the reference in the example.